### DISCUSSION TITLE
New leadership and business model

### DISCUSSION THREAD

**Discussion by tomdoeslinux** [tomdoeslinux/PixaPencil#277] [Announcements]
Hello 👋,

I (Tom) am the new lead developer for this app. Our next step is to overhaul the app by rewriting it in React Native.

Please note, there will be an EULA in place and we have shifted to a **for-profit business model**, but we are still dedicated to making the app open source.

**Comment by searinminecraft** [comment]
with the EULA in place, i dont think i would consider this an open source project anymore, because it clearly states you're not allowed to modify and distribute it, even use the app for commercial purposes

i would rather wait until someone dedicated forks the app based on the last gplv3 commit, just like the libresprite devs did for aseprite.

**Reply by tomdoeslinux** [reply]
The app isn't popular enough to be forked. 

Overall, as the new owner we are looking to make a profit out of this app while also remaining open source. 

If it bothers you then don't use the app.
